google/cloud/spanner_v1/pool.py

```diff
@@ -229,12 +260,24 @@ def get(self, timeout=None):
         if timeout is None:
             timeout = self.default_timeout
 
+        start_time = time.time()
+        current_span = get_current_span()
+        span_event_attributes = {"kind": type(self).__name__}
+        current_span.add_event("Acquiring session", span_event_attributes)
         session = self._sessions.get(block=True, timeout=timeout)
+        span_event_attributes["session.id"] = session.session_id
```

Comment:
Here `session.session_id ` can be empty, if session does not exist. Move this attribute addignment to after the if block below, as it ensures that session will be available.